services/dask-sidecar/src/simcore_service_dask_sidecar/settings.py

```diff
@@ -43,6 +43,10 @@ class Settings(BaseCustomSettings):
 
     # dask config ----
 
+    DASK_CLUSTER_ID_PREFIX: Optional[str] = Field(
+        "CLUSTER_", description="This defines the cluster name prefix"
+    )
```

Comment:
MINOR: in all examples, your prefix end with ``_``. Is this a **pre-requisite** so you can separate the ID from the prefix later on? If so, add a validation regex like ``^[A-Z0-9]+_$`` (double check pls)

Reply:
no there is no such constraint. in can be anything.